dev_guide/event_notifications.adoc

```diff
@@ -0,0 +1,155 @@
+[[dev-guide-event-notifications]]
+= Event Notifications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+For each of your projects, you can choose to receive email notifications about
+various xref:../dev_guide/events.adoc#dev-guide-events[events], including:
+
+* _Dead deployment_: A failed deployment in which no other deployment in the
+rollout history is able to remain active. The related `deploymentConfig` has no
+deployments with active replicas.
+
+* _Failed deployment_: A new deployment fails and another deployment in the
+ rollout history is now active. The related `deploymentConfig` has one active
+ deployment, but it is not the latest deployment.
+
+* _Dead build_: Occurs when automatically triggered builds are failing for a
+specified `buildConfig` that previously had successful builds.
+
+* _Dead persistentVolumeClaim_: A claim that is in the pending state for longer
+than some reasonable amount of time. These are claims whose resource requests
+likely do not match with any existing persistent volumes in the cluster.
+
+* _Lost persistentVolumeClaim_: A claim that is bound to a backing persistent
+volume, but the persistent volume has unexpectedly gone away.
+
+Notifications are sent to the email address of the creator of the project.
```

Comment:
It may be useful to explain how we determine the project creator - we use the "requester" field of the project. (Do we show/refer to that as "project owner" in our UI?)